src/app/harbor/battles/battles.tsx

```diff
@@ -166,7 +166,10 @@ export default function Matchups({ session }: { session: HsSession }) {
       if (currentStep >= totalSteps) {
         clearInterval(spinInterval)
         document.body.style.transform = 'rotate(0deg)' // reset to initial state
-        document.body.style.paddingTop = '1px'
+        document.body.style.paddingTop = '0'
```

Comment:
Just a question deleting this line may work why we are setting it to 0  ?

Reply:
Same for the others.

Reply:
Actually we don't need to set it to 0, I was just testing stuff and left it - transform and willChange is enough

Reply:
But it doesn't change anything, as it's initially 0 too, so Ig it's up to reviewer to leave it or not ; )

Reply:
Setting it to 1px fixed the white blocks issue before which was why it was included FYI